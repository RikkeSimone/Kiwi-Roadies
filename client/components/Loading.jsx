import React from 'react'
import { connect } from 'react-redux'

class Loading extends React.Component {
  render () {
    return (
      <div className="loading-screen-image">
        <img src="./graphics/loadingscreen-revisioned.png" className="kiwi-car"/>
      </div>
    )
  }
}

export default connect()(Loading)
