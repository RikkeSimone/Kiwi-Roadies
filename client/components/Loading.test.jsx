import React from 'react'
import { renderWithRedux } from '../testing/utils'
import '@testing-library/jest-dom'
import 'regenerator-runtime/runtime'

import Loading from './Loading'

test('Loading renders driving kiwi', async () => {
  const { asFragment } = renderWithRedux(<Loading />)
  expect(asFragment()).toMatchSnapshot()
})
