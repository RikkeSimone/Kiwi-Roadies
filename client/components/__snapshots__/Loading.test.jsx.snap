// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders driving kiwi 1`] = `
<DocumentFragment>
  <div
    class="loading-screen-image"
  >
    <img
      class="kiwi-car"
      src="./graphics/loadingscreen-revisioned.png"
    />
  </div>
</DocumentFragment>
`;
